onnxruntime/core/providers/shared_library/provider_interfaces.h

```diff
@@ -665,6 +665,10 @@ struct ProviderHost {
   virtual bool Graph__GetInitializedTensor(const Graph* p, const std::string& tensor_name, const ONNX_NAMESPACE::TensorProto*& value) = 0;
 
   virtual const Node* Graph__ParentNode(const Graph* p) const = 0;
+  virtual const Graph* Graph__ParentGraph(const Graph* p) const = 0;
+  virtual const std::string& Graph__Name(const Graph* p) noexcept = 0;
```

Comment:
I suggest make it: 
`virtual const std::string& Graph__Name(const Graph* p) const noexcept = 0;`
to be aligned with ORT graph API.
Same for `virtual const std::vector<const NodeArg*>& Graph__GetInputsIncludingInitializers(const Graph* p) noexcept = 0;`

Reply:
Thanks for pointing that out. Those have been included in the 2nd commit